boards/xtensa/esp32/doc/esp32.rst

```diff
@@ -20,7 +20,7 @@ From Wikipedia:
 The features include the following:
 
 - Dual core Xtensa microprocessor (LX6), running at 160 or 240MHz
-- 520KiB of SRAM
+- 520KB of SRAM
```

Comment:
I confess this is the first I've heard of the term "kibibytes".  This use of KiB vs. KB is the only use in all our documentation.  While you're right, we have many (if not all) uses of KB meaning 1024 bytes not 1000 bytes since that's the generally accepted "historical" meaning of KB.  So, I'd recommend using KB here, consistent with our use of KB throughout the docs.